src/Microsoft.Graph.Core/Requests/GraphClientFactory.cs

```diff
@@ -113,9 +113,10 @@ public static HttpClient Create(string version = "v1.0", string nationalCloud =
         /// <returns></returns>
         public static IEnumerable<DelegatingHandler> CreateDefaultHandlers()
         {
-            featureFlags = FeatureFlag.RetryHandler | FeatureFlag.RedirectHandler;
+            featureFlags = FeatureFlag.CompressionHandler | FeatureFlag.RetryHandler | FeatureFlag.RedirectHandler;
```

Comment:
FYI. Not specifically applicable to this PR, but we should have an extension method for adding FeatureFlag values to the bitfield. If featureFlags was previously set with other flags, we'd be clearing them here. Tracked with #439

Reply:
@MIchaelMainer Correct. This will be handled as part of #439 & #430. I propose each DelegatingHandler [register](https://github.com/microsoftgraph/msgraph-sdk-dotnet/blob/po/telemetryHandler/src/Microsoft.Graph.Core/Requests/Middleware/RetryHandler.cs#L60) it's feature flag with the request context and let telemetry handler take the request context featureUsage value and add it to the request header. Here a sneak peak of this [implementation](https://github.com/microsoftgraph/msgraph-sdk-dotnet/blob/po/telemetryHandler/src/Microsoft.Graph.Core/Requests/Middleware/TelemetryHandler.cs).